### PR TITLE
docs: fix dependency and zip issues with Universal guide

### PR DESCRIPTION
**Pull request by brandonroberts**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

The zip file from Angular Universal guide download was missing the `ts-loader` dependency.
Scripts were being copied across example package.json files

### CODE REVIEW

File: aio/tools/example-zipper/customizer/package-json/testing.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "dependencies": [],
   "devDependencies": [
     "@angular/cli",
+    "@angular-devkit/build-angular",
```
Comment (gkalpak) [inline aio/tools/example-zipper/customizer/package-json/testing.json]: Nit: I know we have it like this in other similar files, but typically, `@angular-devkit/...` should be above `@angular/...`.

**Review by gkalpak — COMMENTED** [review]
Can you add more info on what the problem is and how the changes in `packageJsonCustomizer` solve it?
Also, the issue mentioned in the commit message seems unrelated (or am I missing something).

**Comment by gkalpak** [comment]
Also, the first commit should mention #24671, not #24689.
(#24689 should be mentioned in the second commit afaict.)

**Comment by brandonroberts** [comment]
I updated the commit messages. Some backstory on the fix.

Previously, `require` was being used to include the `aio/tools/example-zipper/customizer/package-json/package.json` file. The json was then modified with `scripts`, `dependencies` and `devDependencies` based on the example. Since require only resolves once, the same json was being modified for each example instead of using a new template. This changes provides a "clean" template `package.json` to use for each zip file.

**Comment by gkalpak** [comment]
I see, thx for clarifying 👍

File: aio/tools/example-zipper/customizer/package-json/cli.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
   ],
   "dependencies": [],
   "devDependencies": [
-    "@angular/cli",
     "@angular-devkit/build-angular",
+    "@angular/cli",
```
Comment (gkalpak) [inline aio/tools/example-zipper/customizer/package-json/cli.json]: :100:

File: aio/tools/example-zipper/customizer/package-json/packageJsonCustomizer.js [hunk 3]
```diff
@@ -10,10 +11,11 @@ class PackageJsonCustomizer {
     this.dependenciesPackageJson = require(path.join(examplesPath, '/shared/package.json'));
     this.scriptsPackageJson = require(path.join(examplesPath, '/shared/boilerplate/systemjs/package.json'));
     this.basePackageJson = require(`${packageFolder}/base.json`);
+    this.templatePackageJson = fs.readFileSync(`${packageFolder}/package.json`).toString();
   }
 
   generate(type = 'systemjs') {
-    let packageJson = require(`${packageFolder}/package.json`);
+    let packageJson = JSON.parse(this.templatePackageJson);
```
Comment (gkalpak) [inline aio/tools/example-zipper/customizer/package-json/packageJsonCustomizer.js]: Although not strictly necessary, I would consider making switching all `require(...)` calls to `JSON.parse(fs.readFileSync(..., 'utf8'))`, just to be safe (against future changes).

**Review by gkalpak — APPROVED** [review]

File: aio/tools/example-zipper/customizer/package-json/packageJsonCustomizer.js (hunk 3 above)
Comment (brandonroberts) [reply aio/tools/example-zipper/customizer/package-json/packageJsonCustomizer.js]: Done

**Pull request closed without merging**